docs/src/modules/ROOT/pages/optimization-algorithms/optimization-algorithms.adoc

```diff
@@ -686,21 +684,9 @@ public interface Solver<Solution_> {
 ----
 
 The ``BestSolutionChangedEvent``'s `newBestSolution` may not be initialized or feasible.
-Use the `isFeasible()` method on ``BestSolutionChangedEvent``'s new best `Score` to detect such cases:
-
```

Comment:
Can we keep this snippet?

Reply:
There is no quickstart that would show this. Therefore I decided not to.
We are likely to be refactoring this feature in the short term anyway, so the docs would be rewritten.